### ISSUE TITLE
Decentralised CFP selection in Conferences

### ISSUE THREAD

**Issue by aj07** [Blockternship/projects#4]
**Overview**
Hy!! This is Ankit from New Delhi. I am a big fan of decentralized platform and wanted to build something for the community by Blockternship.

I have been coding and learning lot these days about Ethereum and Dapp ecosystem. 

**Overview &Philosophy**
There are tons of conferences happening everywhere across the globe.  Most of the time speakers are selected those who are known to organizers. It would be great if speakers are selected by the attendees from a list given by organizers. This would make conference more transparent and useful. Selection of speaker should be unbiased and attendees focused. 

Outcome should be fully functional dapp with functional UI/UX. If time and gas fee permits we will include is ethdns also to have uniqueness among identity. 

**Next Step**
Looking forward some inputs from community and green flag to start the project.
Also looking for UI developer who can help me in building this as proper usable dapp.

Github username: [aj07](https://github.com/aj07)
Twitter: [a4nkit ](https://twitter.com/a4nkit)

**Comment by Quazia** [comment]
Seems like it might be a good use case for general TCR technology?

**Comment by YalorMewn** [comment]
Hey @aj07 could you explain CFP a little more, first explain how that tech works?

*Then when you say " It would be great if speakers are selected by the attendees from a list given by organizers." 

- Do you mean a pre- defined list created by organizers or a list that is generated by attendees of the conference? 

- Then is it a vote system? How do you envision this working? 

Also how you plan on aggregating the data given from organizers ?

**Comment by YalorMewn** [comment]
Hey @Quazia could you share the meaning of TCR technology ?

**Comment by aj07** [comment]
@YalorMewn , yes there will be the list of speakers suggested by admin. From there, cofirmed attenddes will choose whom they want to listen.

Yes, this will be bit similar to vote system where top speakers are selected on the basis of vote they got.
Will you please let me know more about the question "Also how you plan on aggregating the data given from organizers ?"

**Comment by Quazia** [comment]
“Dev Diary II: A Walkthrough of the adChain Registry TCR in Solidity” @ad_chain https://medium.com/metax-publication/a-walkthrough-of-the-adchain-registry-tcr-in-solidity-72addcde17fb

Token curated registry. Seems like a good tech fit for the problem you're trying to solve here.

**Comment by deamme** [comment]
I think this type of project is the right size for the Hackathon, so I would approve. Some questions though:
- How do you handle voting for different conferences?
- Do conferences need to generate their own site and deploy their own contract?

**Comment by aj07** [comment]
@deamme thnaks! for asking question.
For each conferences each user can vote only single time. Multiple votings aren’t allowed. We can link ens also for making things easy. 

Conferences admin will request on our dapp to start the process by filling up some details. After that they can request attendess to vote for the speaker on our platform. Voting will be done on our platform.  Conferences admin will only request attendees to visit our dapp and vote.

P.S:  we can add functionality (conferences charge) also, if project mentor will ask us to do so.

**Comment by deamme** [comment]
@aj07 Thanks for the clarification. This is definitely doable to have everything on a single site.

**Comment by aj07** [comment]
Thanks!! @deamme

**Comment by YalorMewn** [comment]
Welcome to the "Blockternship" @aj07 Ankit, look forward to seeing this come together!

**Comment by aj07** [comment]
Thanks @YalorMewn and other teammates. I will start working on this.

**Comment by YalorMewn** [comment]
One last thing @aj07 , do you have a repo yet or should we start a fresh one for you here ?

**Comment by aj07** [comment]
@YalorMewn please, start the repo. I will submit my code there

**Comment by YalorMewn** [comment]
You're all set up [here](https://github.com/Blockternship/shiny-telegram), with admin access so change the name whenever you decide it's appropriate or keep shiny- telegram 😂

**Comment by aj07** [comment]
Hey @YalorMewn . Thanks for that.  :) I will start working on it.
I changed it DXcon (thought to give decentralized feeling )

**Comment by YalorMewn** [comment]
Hey @aj07 I would like to invite you to the first call of the hackathon, would you send me a quick email here YalorTackson@gmail.com also you can add yourself to the event [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=NjlxM2dtdDd2bHA5ODNxbzNsYm5vOTMyZDkgeWFsb3J0YWNrc29uQG0&tmsrc=yalortackson%40gmail.com)

**Comment by andytudhope** [comment]
This looks good. Happy to comment and review in that repo when there is code submitted.

**Comment by smitrajput** [comment]
Would be more than happy to join as a teammate.

**Comment by aj07** [comment]
cool  @smitrajput Happy to work with you. :)

**Comment by status-open-bounty** [comment]
Balance: 0 ETH
Tokens: DAI: 515.00
Contract address: [0xe4ed0aee83a22124711d3d989563945efac54e78](https://etherscan.io/address/0xe4ed0aee83a22124711d3d989563945efac54e78)
Network: Mainnet
Paid to: aj07
Visit [https://openbounty.status.im](https://openbounty.status.im) to learn more.

**Comment by codehard-achal** [comment]
I'd love to be a contributing member on this project @aj07 . Keep me in the loop and do instruct me about what all needs to be done. Let's get this going!

**Comment by aj07** [comment]
@status-open-bounty Done from my side. @smitrajput Please do from your side too

**Comment by smitrajput** [comment]
@status-open-bounty All set!

**Comment by aj07** [comment]
@YalorMewn Can we have 3 member in team?

**Comment by YalorMewn** [comment]
As many as you like

**Comment by YalorMewn** [comment]
Thanks for participating, don't forget to collect you bounty !

**Comment by aj07** [comment]
Thanks @YalorMewn
